src/base/configuration.cpp

```diff
@@ -168,6 +168,11 @@ std::string Configuration::copyright() {
          << "  See http://github.com/timothy-king/glpk-cut-log for copyright"
          << " information\n\n";
     }
+    if (Configuration::isBuiltWithCoCoA()) {
```

Comment:
I think the outer if misses a guard for `Configuration::isBuiltWithCoCoA()`.

Reply:
Good catch!